guide/02-api-overview/release_notes_210.ipynb

```diff
@@ -0,0 +1,897 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c34e7960",
+   "metadata": {},
+   "source": [
+    "# What's new in version 2.1.0\n",
+    "\n",
+    "## New\n",
+    "* Adds [Dask](https://www.dask.org/) Support\n",
+    "  * advanced parallel processing for analytics\n",
+    "  * see [`from_parquet()`](/python/api-reference/arcgis.features.toc.html#arcgis.features.GeoAccessor.from_parquet)\n",
+    "* Adds descriptive information for results of asynchronous functions\n",
+    "  * see a `future` parameter for functions and methods in the documentation\n",
+    "* Adds hyperlinks to return values in API Reference documentation\n",
+    "* Removes `ntlm_authentication` and replaces with `gsspi/kerberos` on non-Windows machines\n",
+    "\n",
+    "\n",
+    "### [Guides](../)\n",
+    "* [Install and Setup](../guide/install-and-set-up/)\n",
+    "  * Adds [`Install in Google Collab`](../guide/install-and-set-up/#install-in-google-colaboratory) section\n",
+    "* [Deep Learning with ArcGIS](../guide/)\n",
+    "  * [2d Computer Vision]\n",
+    "    * [Pixel Classification]\n",
+    "      * [How PSETAE works](../guide/how-PSETAE-works)\n",
+    "  \n",
+    "### [Samples](/python/samples/)\n",
+    "* [GIS analysts and data sciencists](/python/samples)\n",
```

Comment:
scientists

Reply:
Hey Noah - it's a pre-deployment thing. The structure of the hyperlinks is all through relative links so just running the stand alone notebook most links will not work. Most of them will work if you build the site locally.

Reply:
Fixed.

Reply:
In addition, the relative links to the api reference are structured in a way that they will not work even when building the site locally. After clicking on the link, uou have to remove the word `python` from the url in order for the link to work in the local site.